core/src/test/groovy/jenkins/security/HexStringConfidentialKeyTest.groovy

```diff
@@ -1,43 +0,0 @@
-package jenkins.security
-
-import org.junit.Rule
-import org.junit.Test
-
-/**
- *
- *
- * @author Kohsuke Kawaguchi
- */
-class HexStringConfidentialKeyTest {
-    @Rule
-    public ConfidentialStoreRule store = new ConfidentialStoreRule()
-
-    @Test
-    void hexStringShouldProduceHexString() {
-        def key = new HexStringConfidentialKey("test",8)
-        assert key.get() =~ /[A-Fa-f0-9]{8}/
-    }
-
-    @Test
-    void multipleGetsAreIdempotent() {
-        def key = new HexStringConfidentialKey("test",8)
-        assert key.get()==key.get()
-    }
-
-    @Test
-    void specifyLengthAndMakeSureItTakesEffect() {
-        [8,16,32,256].each { n ->
-            new HexStringConfidentialKey("test"+n,n).get().length()==n
```

Comment:
Presumably forgotten `assert`; Groovy does not grok any difference between an expression and a statement so it would compile as is.

Reply:
Seems like yet another reason to avoid groovy for anything important